src/Bicep.Core.IntegrationTests/DecoratorTests.cs

```diff
@@ -30,7 +30,7 @@ public void ParameterDecorator_MissingDeclaration_ExpectedParameterDeclaration()
             using (new AssertionScope())
             {
                 template.Should().NotHaveValue();
-                diagnostics.Should().HaveDiagnostics(new[] {
+                diagnostics.ExcludingLinterDiagnostics().Should().HaveDiagnostics(new[] {
```

Comment:
Would it be better to use this pattern:

   diagnostics.Should().HaveCompilerDiagnostics(new[] {

?